backend/app.py

```diff
@@ -7,39 +7,56 @@
 import os
 from pathlib import Path
 from subprocess import call as sp_call
-from typing import Any, Dict, List, Union, Callable, Set
+from typing import Any, Dict, List, Union, Set
 from functools import cache
 
 import numpy as np
 import pandas as pd
 import uvicorn
-from fastapi import FastAPI, Query, Response
+import urllib.parse
+from fastapi import FastAPI, Query
 from fastapi.middleware.cors import CORSMiddleware
 from fastapi.middleware.gzip import GZipMiddleware
 from pydantic import BaseModel
-import urllib.parse
 
+from enclave_wrangler.config import OUTDIR_DATASETS_TRANSFORMED, OUTDIR_OBJECTS
 from enclave_wrangler.dataset_upload import upload_new_container_with_concepts, upload_new_cset_version_with_concepts
 from enclave_wrangler.datasets import run_favorites as update_termhub_csets
 from enclave_wrangler.new_enclave_api import make_read_request
 
+
 DEBUG = True
 PROJECT_DIR = Path(os.path.dirname(__file__)).parent
-CSV_PATH = f'{PROJECT_DIR}/termhub-csets/datasets/prepped_files/'
+# GLOBAL_DATASET_NAMES = list(FAVORITE_DATASETS.keys()) + ['concept_relationship_is_a']
+GLOBAL_DATASET_NAMES = [
+    'concept_set_members',
+    'concept',
+    'concept_relationship_subsumes_only',
+    'concept_set_container',
+    'code_sets',
+    'concept_set_version_item',
+    'deidentified_term_usage_by_domain_clamped',
+    'concept_set_counts_clamped'
+]
+GLOBAL_OBJECT_DATASET_NAMES = [
+    'researcher'
+]
 DS = None  # Contains all the datasets as a dict
-ds = None  # Contains all datasets, transformed datasets, and a few functions as a nameespace
+DS2 = None  # Contains all datasets, transformed datasets, and a few functions as a namespace
```

Comment:
Rename: `ds` -> `DS2`. Capitalization is standard convention for global constants in Python. This helps me as well because I can look at it and know it's global.

After doing this session today, I would like to move forward with the RDBMS as a next step or very soon step if there is nothing else in the way that is demanding. Needing to restart the server a lot when debugging the backend is definitely slowing development down, and it looks like I won't be able to get a new laptop for a long time to come.

Reply:
Making first steps on concept set update to enclave and getting app deployed are the top priorities. I don't see update on the In process/sprint list. It should be there.

I think RDBS would be great. Can you give some estimate of effort before starting?